01-js-intro-and-jquery/08/js/scripts.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+	
+	var alias = '';
+	
+	var focusAlias = function() {	
+		$('.alias').focus(); 
```

Comment:
This $('.alias') is used here and also in line 10, maybe you can create a variable to cached this up, just to avoid 2 searchs into the DOM.